### PR TITLE
PMM 2.11 changes

### PR DESCRIPTION
**Pull request by AlekSi**

### CODE REVIEW

**Comment by AlekSi** [comment]
Lost PR. Not sure if it is needed or not.

**Comment by JiriCtvrtka** [comment]
It is pmm-api-tests part for pg_stat_monitor. So it should be merged too.

File: Gopkg.toml [hunk 1]
```diff
@@ -10,4 +10,4 @@ required = [
 
 [[constraint]]
   name = "github.com/percona/pmm"
-  branch = "PMM-2.0"
+  branch = "release/2.11"
```
Comment (JiriCtvrtka) [inline Gopkg.toml:13]: @AlekSi Should I revert it to PMM-2.0? Thanks.

Comment (BupycHuk) [reply Gopkg.toml:13]: yes

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**